tests/rhosak/kas_with_instance.spec.ts

```diff
@@ -65,8 +65,8 @@ const filterByName = async function (page, name, skipClick = false) {
 
   await page.locator('button[role="option"]:has-text("Name")').click();
 
-  await page.getByRole('searchbox', { name: 'Filter by name' }).click();
-  await page.getByRole('searchbox', { name: 'Filter by name' }).fill(name);
+  await page.getByTestId('large-viewport-toolbar').getByPlaceholder('Filter by name').click();
```

Comment:
Im not sure but we decided to do not use testId if I'm not wrong. @andreaTP

Reply:
maybe you could use something like `page.locator(searchbox, { hasText: Filter by name} )` ? Of course with proper element type and text

Reply:
I know that this seachbox is a little problematic. There are two elements that will be found in the DOM, one is hidden thought and ?should be displayed on small devices? 🤔. 
Perhaps there is some method or option to force it here to look only for visible elements or something. 
![image](https://user-images.githubusercontent.com/20477661/216002864-e5784559-e527-495d-b01b-ccc4ba81c832.png)

@asanzgom please look into the documentation.

Reply:
@mmusil what documentation are you talking about?

Reply:
@riccardo-forina this is an already existing test. Do you suggest us to remove all 'filterBy' tests?

Reply:
> @riccardo-forina this is an already existing test. Do you suggest us to remove all 'filterBy' tests?

No, I think he was just suggesting looking for how the same challenge has been solved in code that has already been written (in this case in the unit tests).

But no, there is no suggestion not to write e2e tests because unit tests do already exist. Putting it explicitly here just to be sure this is clear and understood.

Reply:
@kornys UI unit tests are relying on dataTestId: https://github.com/redhat-developer/app-services-ui-components/blob/d91d0709b8ca6c534a6997440823479f68c43b09/src/shared/TableToolbar/ChipFilter/ChipFilter.test.tsx#L22